interface-definitions/load-balancing-inbound-ipvsadm.xml.in

```diff
@@ -0,0 +1,195 @@
+                    <format>u32:0-65535</format>
+                    <description>Number for this rule</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 0-65535"/>
+                  </constraint>
+                  <constraintErrorMessage>Back-end server load balancing rule numbers are not between 0 and 65535</constraintErrorMessage>
+                </properties>
+                <children>
+                  #include <include/generic-disable-node.xml.i>
+                  <leafNode name="protocol">
+                    <properties>
+                      <help>Load balancing protocol</help>
+                      <completionHelp>
+                        <list>tcp udp sctp</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>tcp</format>
+                        <description>TCP Protocol</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>udp</format>
+                        <description>UDP Protocol</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>sctp</format>
+                        <description>SCTP Protocol</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>^(tcp|udp|sctp)$</regex>
+                      </constraint>
+                      <constraintErrorMessage>The load balancing protocol must be one of tcp, udp, and sctp</constraintErrorMessage>
+                    </properties>
+                    <defaultValue>tcp</defaultValue>
+                  </leafNode>
+                  <leafNode name="virtual-address">
+                    <properties>
+                      <help>Virtual server IP</help>
+                      <valueHelp>
+                        <format>ipv4</format>
+                        <description>IPv4 VIP for the virtual server</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ipv6</format>
+                        <description>IPv6 VIP for the virtual server</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ip-address"/>
+                      </constraint>
+                      <constraintErrorMessage>Virtual address must be a valid IPv4 or IPv6 address (e.g. 192.0.2.3 or 2001:db8:ff::10)</constraintErrorMessage>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                  #include <include/port-number.xml.i>
+                  <leafNode name="mode">
+                    <properties>
+                      <help>Execute load balancing operation mode</help>
+                      <completionHelp>
+                        <list>dr tun nat</list>
```

Comment:
Can we name this `direct`, `tunnel`, `nat` instead?

Reply:
done